public/index.html

```diff
@@ -1,4 +1,7 @@
 <!DOCTYPE html>
+<style>
```

Comment:
We can take out Maitree @alyssabull

Reply:
Yes - sounds good. I think the Heebo font is sufficient.